openedx/stanford/cms/urls.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from django.conf.urls import patterns
+from django.conf.urls import url
+
+
+urlpatterns = patterns(
+    'contentstore.views',
+    url(r'^settings/send_test_enrollment_email/{}$'.format(settings.COURSE_KEY_PATTERN), 'send_test_enrollment_email', name='send_test_enrollment_email'),
+    url(r'^utilities/{}$'.format(settings.COURSE_KEY_PATTERN), 'utility_handler'),
+    url(r'^utility/captions/{}$'.format(settings.COURSE_KEY_PATTERN), 'utility_captions_handler'),
+    url(r'^utility/bulksettings/{}$'.format(settings.COURSE_KEY_PATTERN), 'utility_bulksettings_handler'),
+)
+urlpatterns += (
```

Comment:
shouldn't these urls be behind a Feature Flag?

Reply:
Added